scss/mixins/_banner.scss

```diff
@@ -0,0 +1,10 @@
+@mixin bsBanner($file, $suffix:"") {
+  /*!
+   * Bootstrap #{$file} v5.2.0-beta1 (https://getbootstrap.com/)
+   * Copyright 2011-2022 The Bootstrap Authors
+   * Copyright 2011-2022 Twitter, Inc.
+   * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+   * #{$suffix}
```

Comment:
Let's nix the suffix line and argument entirely. It's for one file, and we have this mentioned in our docs.

```suggestion
```